modules/ROOT/pages/design-common-errors-08.adoc

```diff
@@ -0,0 +1,360 @@
+= Common Errors in Conforming to RAML 0.8
+
+When testing its new RAML parser (which was released on October 24, 2018), MuleSoft identified ten common errors in API specifications written in RAML 0.8. This topic explains the errors with examples and describes how to fix them. They are not listed in any particular order, such as frequency of occurrence in the sample of API specifications that MuleSoft tested.
+
+If you open in the code editor a published API specification that is written in RAML 0.8 and you see error indicators, this topic might help you to resolve most of the errors. If you see errors that this topic does not mention, refer to the specification for RAML 0.8, which you can read on https://raml.org/[raml.org].
+
+== Common Error 1
+// APIMF-824
+
+*_Not using a data type of string for the example of a Boolean enum_*
+
+If the values for a Boolean enum are either both strings or both Boolean values, the example of the enum must be a string. To illustrate the error, this snippet shows an example using the wrong data type.
+
+----
+#%RAML 0.8
+title: ExampleRAML
+/nob:
+  head:
+    responses:
+      204:
+        headers:
+          X-NOB-Exists:
+            enum: [true, false]
+            example: true
+----
+
```

Comment:
I would separate the cases, first just the incorrect and correct example and then the same for the enum facet. Clarify that both the enum and example facets are validated in the same way.

Reply:
What do you mean by "the enum and example facets are validated in the same way"?

Reply:
When you have an enum facet in a type definition, any valid payload must be one of the instances present in the array of enums. So, each instance present in the enum should be a valid payload too.

Reply:
If the type is a string type, then yes, the true and false keywords must be quoted to explicit said that is a string with that text and not the boolean representation. If the type is defined for a boolean data type, then it would be wrong quote the keywords.
In this case, the example should be quoted, as the enum values.

Reply:
Resolved with Jonathan's help.